src/components/Editor/index.js

```diff
@@ -141,6 +147,10 @@ class Editor extends React.Component {
             <span className="editor-button-text">&nbsp;&nbsp;{this.state.saveText}</span>
           )}
         </Button>
+
+        <Button className="mx-2" color="success" size="lg" onClick={this.handleDownload}>
+          <FontAwesomeIcon icon={faDownload} />
+        </Button>
```

Comment:
I think the spacing for the button is a bit off (the x-margin is larger between this and the save code option compared to the save code button and the pill) - could you fix that?

Reply:
Would you rather me adjust the margins for the radio button? That would make it more consistent with the buttons in the output pane

Reply:
I went ahead and did that: here is the result

![buttons](https://user-images.githubusercontent.com/10734915/68269050-f7394a80-001d-11ea-9bee-b9fbb2679afe.png)


It's quite difficult to change the margins on those buttons- they have a big fat `!important` next to them

Reply:
Hm, it still feels a bit off - but I think this is good for now. Looks like we might have to add another button in a moment so we'll see how this all works out. Solid work!